votes/initiateNewVote/_EDIT_ME.yml

```
@@ -35,10 +35,18 @@ canShuffleCandidates: true
 # 5. Insert here a short description of the vote objectives and link to the
 #    issue it was discussed on to give the full context.
 footerInstructions: |
-  TBD
+  The objective of this vote is to decide whether duration to determine
+  inactivity should be reduced from 12 months to 9 months. This vote was discussed
+  in pull-request https://github.com/nodejs/node/pull/52459 and the issue #1524.
 
 # 6. Optionally, insert a brief introduction for the vote PR, in the markdown format.
 prBody: |
+  Currently, Node.js collaborators are considered inactive if they have not
+  contributed to the project for 12 months. The definition of contribution in
+  the context of Node.js collaborators is any approval or review of a pull
+  request, or any commit to the Node.js repository. The proposed reduction
+  in the duration of inactivity for Node.js collaborators, from 12 months
+  to 9 months.
```

I think the current wording is already pretty clear about the specific criteria but if there are still concerns about the definition of contribution not being super duper clear, we can go with

```suggestion
  Currently, Node.js collaborators are considered inactive if in the
  past 12 months, they haven't done any of the following:
  
  - Opening any landed pull request
  - Approving any landed pull request
  - Landing any commit to the Node.js repository
  
  This vote is to decide whether we want to reduce the timespan from 12 months to
  9 months.
```